python_lib/qss_compiler/examples/pulse.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Using this notebook requires manual install of MLIR python bindings:
+# 1. Copy or link to `qss-compiler/third_party/mlir/pyproject.toml` from `~/.conan/data/llvm/qss/stable/<hash>/python_packages/mlir_core`  # noqa: E501
+# 2. Run `pip install -e .`
+# 3. Copy or link to `~/.conan/data/llvm/qss/stable/<hash>/python_packages/mlir_core/mlir/_mlir_libs/libMLIRPythonCAPI.dylib` from `<python>/lib`  # noqa: E501
+# 4. build compiler
+# 5. cd `qss_compiler/python_lib`
+# 6. run `bash setup_mlir.sh`
+# 7. run `pip install -e .`
+
+from qss_compiler.mlir.ir import Context, InsertionPoint, Location, Module
+from qss_compiler.mlir.ir import F64Type, IntegerType, IndexType, ComplexType
+from qss_compiler.mlir.dialects import arith, func, complex
+
+from qss_compiler.mlir.dialects import pulse, quir, qcs
+
+import numpy as np
+
+
+num_samples = 3
+samples = np.arange(0, num_samples) + np.arange(0 + 1, num_samples + 1) * 1j
+samples_2d = np.zeros((samples.shape[0], 2))
+samples_2d[:, 0] = np.real(samples)
+samples_2d[:, 1] = np.imag(samples)
+
+
+with Context() as ctx:
+
+    pulse.pulse.register_dialect()
+    quir.quir.register_dialect()
+
+    f64 = F64Type.get(ctx)
+    i32 = IntegerType.get_signless(32)
+    i1 = IntegerType.get_signless(1)
+
+    idx = IndexType.get()
+
+    mf = pulse.MixedFrameType.get(ctx)
+    wf = pulse.WaveformType.get(ctx)
+
+    with Location.unknown(ctx) as loc:
+        module = Module.create(loc)
+
+        with InsertionPoint(module.body):
+            function = func.FuncOp("main", ([], [i32]))
+            function.add_entry_block()
+
+        with InsertionPoint(module.body):
+
+            seq1 = pulse.SequenceOp("seq_1", [wf, wf, mf, mf], [i1])
+            seq1.add_entry_block()
+
+        #     with InsertionPoint(seq1.entry_block):
```

Comment:
Should these be un-commented or removed?

Reply:
Removed in https://github.com/openqasm/qe-compiler/pull/264/commits/02bafaedd12067bd0eaffd7566ee37306f01e303